src/rustdoc_schema.graphql

```diff
@@ -55,6 +55,36 @@ interface Item {
   span: Span
 }
 
+"""
+https://docs.rs/rustdoc-types/0.11.0/rustdoc_types/struct.Item.html
+https://docs.rs/rustdoc-types/0.16.0/rustdoc_types/struct.Module.html
+"""
+type Module implements Item & Importable {
+  # properties from Item
+  id: String!
+  crate_id: Int!
+  name: String
+  docs: String
+  attrs: [String!]!
+  visibility_limit: String!
+
+  # own properties
+  is_crate: Boolean!
```

Comment:
It's possible that we don't even want to consider the top-level module (the one with is_crate=true) to be an item? But I think it's okay.

Reply:
I think we probably do want to consider it a module, since that both matches how Rust thinks about it and also because otherwise it'll be hard (perhaps even impossible) to answer "what items are at top level in the crate."